src/App.js

```diff
@@ -1,33 +1,45 @@
-import React, { useEffect, useState } from 'react';
-import './App.css';
+import React, { useEffect, useState } from "react";
+import "./App.css";
```

Comment:
Can you change the double quotes back to single quotes. I'd like to try and keep the styling uniform with the rest of the codebase. Generally I just use double quotes for component props. Other than that, the rest of the changes look great!

Reply:
Sure @devcshort, let me do that. Thanks for pointing this out!

Reply:
Thanks for the update! Merging now 🙂

Reply:
@devcshort Glad could contribute :smile: 

I missed 2-3 changes for the commas, hope that's ok.

Reply:
I think that's fine, it looks great

Reply:
Cool, thanks!